src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -27,24 +28,21 @@
 {
     protected function listBundles(OutputInterface $output)
     {
-        $output->writeln('Available registered bundles with their extension alias if available:');
-
-        if (class_exists('Symfony\Component\Console\Helper\Table')) {
-            $table = new Table($output);
-        } else {
-            $table = $this->getHelperSet()->get('table');
-        }
-
-        $table->setHeaders(array('Bundle name', 'Extension alias'));
+        $headers = array('Bundle name', 'Extension alias');
+        $rows = array();
         foreach ($this->getContainer()->get('kernel')->getBundles() as $bundle) {
             $extension = $bundle->getContainerExtension();
-            $table->addRow(array($bundle->getName(), $extension ? $extension->getAlias() : ''));
+            $rows[] = array($bundle->getName(), $extension ? $extension->getAlias() : '');
         }
 
-        if (class_exists('Symfony\Component\Console\Helper\Table')) {
-            $table->render();
+        $message = 'Available registered bundles with their extension alias if available:';
+        if ($output instanceof OutputStyle) {
```

Comment:
This method is typehinted `OutputInterface`. As it might eventually be used by other bundles, not using SymfonyStyle nor custom `StyleInterface` implementations, we need to do this check in order to chose the way to render the table.

Unless we judge other bundles should not extend this class, so we could typehint this method with the `StyleInterface`.

BTW, I don't remember why I used `OutputStyle` instead of `StyleInterface` :/